_data/companies/intuit.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: Intuit
-wfh: Strongly Encouraged
+wfh: Required
 travel: Restricted
 visitors: Restricted
 events: Restricted
-last_update: 2020-03-12
+last_update: 2020-04-13
```

Comment:
Is that a typo or do you have a time machine?

Reply:
I interpreted this as the day that the policy changed. I will change it to today.